tests/operations.go

```diff
@@ -766,6 +766,26 @@ func (ops *Operations) GetHealthyCSPCount(spcName string, expectedCSPCount int)
 	return cspCount
 }
 
+// GetCSPIResourceCount gets healthy csp based on spcName
+func (ops *Operations) GetCSPIResourceCount(cspcName string, expectedCSPICount int) int {
```

Comment:
Can we use either `GetCSPICount` or `GetHealthyCSPICount` function instead of adding this new function. The functionality of this function looks similar to `GetCSPICount`.

Reply:
Umm... Bit different I guess. `GetCSPIResourceCount` just gives you the count of cspi irrespective of it being healthy or not.

Reply:
Will do it as discussed.